.eslintrc.js

```diff
@@ -27,6 +27,7 @@ module.exports = {
     'jest',
   ],
   rules: {
+    '@typescript-eslint/no-unused-expressions': 'off',
```

Comment:
We have a lot of usage of this in our codebase

`expression && doSomething()`

Reply:
maybe just configure https://eslint.org/docs/latest/rules/no-unused-expressions#allowshortcircuit instead so we don't lose the other parts of this rule?

(although personally I don't like `expression && doSomething()` and think plain `if` statements are much more readable, but happy to go with team consensus)

Reply:
Makes sense to only configure the allowShortCircuit, will update and yes for this one I'm just going with the current status quo in the repo and not opening any discussions if it should or shouldn't be like that 😅